website/docs/features/dapp-dev-hot-reloading.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Hot releading during dApp development
+---
+
+Taqueria plugins are able to define templates that can be used to generate project components such as contracts, tests, or operations. Templates are exposed via plugins and use the `create` task to create and register the templates output
+
+## Available Templates
```

Comment:
Should we also add in other plugins that allow users to create from a template. If we mention tests we should probably add in the jest plugin here

Reply:
@alexzbusko I have it in my PR - https://github.com/ecadlabs/taqueria/pull/1225